common/src/main/scala/reactST/reactTable/SUITable.scala

```diff
@@ -132,17 +133,17 @@ class SUITable[
 
       val headerCellRender: HeaderCellRender[D, ColumnInstanceD] =
         (props.headerCell: Any) match {
-          case headerCell: TableHeaderCell => _ => headerCell
+          case headerCell: TableHeaderCell => _ => headerCell.copy(as = layout.tag)
           case fn                          => fn.asInstanceOf[HeaderCellRender[D, ColumnInstanceD]]
         }
 
       val bodyCellRender: BodyCellRender[D] = (props.cell: Any) match {
-        case cell: TableCell => _ => cell
+        case cell: TableCell => _ => cell.copy(as = layout.tag)
```

Comment:
Does this convert the table tags to something else?

Reply:
There's a typleclass mechanism in place in `SUITable` to render HTML table elements or div blocks depending whereas the react-table builder has a layout plugin.